src/desktop/config.coffee

```diff
@@ -64,6 +64,7 @@ module.exports =
   GEOIP_ENDPOINT: 'https://artsy-geoip.herokuapp.com/'
   GOOGLE_ADWORDS_ID: null
   GOOGLE_MAPS_API_KEY: null
+  HASHTAG_LAB_ADS_ENABLED: ['ashley.jelks@artsymail.com', 'luc@artsymail.com', 'junyao@artsymail.com']
```

Comment:
Sure y'all want to encode your email addresses into the default (which is OSS)? Can pass these in via `.env`. Either way!

Reply:
I actually had my non-Artsy email hardcoded first. The purpose of this is to add email addresses  of non-Artsy folks, so they can test the ads without having Artsy admin privileges. I think I'll revert the change and test it with my personal email to confirm that it works and then remove it entirely. Thanks for the heads up @damassi !

Reply:
Note that this can also be tested by updating Force's staging environment, via hokusai:

```
hokusai staging env set HASHTAG_LAB_ADS_ENABLED='foo,bar,baz'
```

but note that that would have to then be parsed (`myVar.split(',')`); ENV vars are just strings.

One other thought is `HASHTAG_LAB_ADS_ENABLED` might be better named `HASHTAG_LAB_ADS_ALLOWLIST`, since that implies a list type, vs a boolean.